api/nakadi-event-bus-api.yaml

```diff
@@ -109,6 +109,15 @@ securityDefinitions:
     type: oauth2
     flow: implicit
     authorizationUrl: 'https://auth.example.com/oauth2/tokeninfo'
+    scopes:
```

Comment:
In our scopes do we have only one resourceId `nakadi` and four scopes for it?
Or do we have three resourceIds `nakadi`, `nakadi.event.type` and `nakadi.event.stream` ?